de/events.json

```diff
@@ -1 +1,27 @@
-{}
+{
+  "page-title": "Nächste Events ({{platform}}) - Eric's GoW Tools",
+  "page-desc": "Liste von zukünftigen Events in Gems of War, täglich aktualisiert.",
+
+  "head-name": "Name",
+  "head-start-date": "Startdati,",
+  "head-end-date": "Enddatum",
+  "head-duration": "Dauer",
+  "head-troop-pet-class": "Truppe/Pet/Klasse",
```

Comment:
Minor nitpick: I'd suggest using "Einheit" instead of "Truppe" for "troops" because the both the button and the tab in the troops menu are labelled "Einheit"